content/post/DataScience.md

```diff
@@ -80,18 +80,18 @@ Various tools for Big Data :
 - [Hadoop](https://in.udacity.com/course/intro-to-hadoop-and-mapreduce--ud617)
 - [Hadoop & Map Reduce](https://www.udacity.com/course/intro-to-hadoop-and-mapreduce--ud617)
 
-![Big Data](/img/BigData.jpg)
+![Big Data](/ALiAS_Blog/images/BigData.jpg)
 
 ## Last Advice :
 
 Learning Online is not an easy task, and with all these massive numbers of resources, it is very easy to get side-tracked while learning online. Data Science is a different path other than all fields of Computer Science and at some point in time, you'll be frustrated that you have wasted all your time in learning these and your output is zero. But don't be disheartened by this, Data Science is like a marathon and you'll eventually have to invest a lot of time in this, leaving all other fields. You will feel like your friends and colleagues learning different other things are much more ahead of you. But after all the self-driven education and regularly organized time investment, you'll be eventually at another level than any of your contenders. It won’t be easy but to motivate your own education you will need perseverance and discipline. Then you can break into the data science industry no matter what your situation is.
 
-<!---![Data Science](/img/DS3.jpg) "image needs to be resized"--->
+<!---![Data Science](/ALiAS_Blog/images/DS3.jpg) "image needs to be resized"--->
```

Comment:
Is this change made manually or automatically generated? If manually, do make sure that README has a mention on naming conventions.

Reply:
Actually, when I first deployed, only the frontmatter images were working, not others and when I looked in the page source, those images added /ALiAS_Blog/ and the others did not. Now all of them work.
So I manually changed all of them and I have also mentioned it under instructions for writers.